src/PrestaShopBundle/Controller/Admin/TranslationsController.php

```diff
@@ -55,113 +60,178 @@ public function overviewAction()
         return parent::overviewAction();
     }
 
-    // redirect to the new translation application
-    // before, clean request params
-    private function redirectToTranslationApp(Request $request)
+    /**
+     * Show translations settings page
+     *
+     * @Template("@PrestaShop/Admin/Improve/International/Translations/translations_settings.html.twig")
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))", message="Access denied.")
```

Comment:
if you don't change the message value, you are not required to set the property (I need to update this part of the docs lol)